components.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
+  "style": "new-york",
+  "rsc": false,
+  "tsx": true,
+  "tailwind": {
+    "config": "",
+    "css": "app/style.css",
+    "baseColor": "zinc",
+    "cssVariables": true,
+    "prefix": ""
+  },
+  "aliases": {
+    "components": "@/components",
+    "utils": "@/lib/utils",
+    "ui": "@/components/shadcn-ui",
+    "lib": "@/lib",
+    "hooks": "@/hooks"
+  },
```

`lib/`や`util/`などもshadcn-uiのためのファイルが生成されるので、全体を`shadcn-ui/`以下に置いた方が良いと思います。
```suggestion
  "aliases": {
    "components": "@/shadcn-ui/components",
    "utils": "@/shadcn-ui/lib/utils",
    "ui": "@/shadcn-ui/components/",
    "lib": "@/shadcn-ui/lib",
    "hooks": "@/shadcn-ui/hooks"
  },
```